spec.html

```diff
@@ -40119,73 +40176,13 @@ <h1>Additional Syntax</h1>
 
     <emu-annex id="sec-additional-syntax-numeric-literals">
       <h1>Numeric Literals</h1>
-      <p>The syntax and semantics of <emu-xref href="#sec-literals-numeric-literals"></emu-xref> is extended as follows except that this extension is not allowed for strict mode code:</p>
-      <h2>Syntax</h2>
-      <emu-grammar type="definition">
-        NumericLiteral ::
-          DecimalLiteral
-          BinaryIntegerLiteral
-          OctalIntegerLiteral
-          HexIntegerLiteral
-          LegacyOctalIntegerLiteral
-
-        LegacyOctalIntegerLiteral ::
-          `0` OctalDigit
-          LegacyOctalIntegerLiteral OctalDigit
-
-        DecimalIntegerLiteral ::
-          `0`
-          NonZeroDigit DecimalDigits?
-          NonOctalDecimalIntegerLiteral
-
-        NonOctalDecimalIntegerLiteral ::
-          `0` NonOctalDigit
-          LegacyOctalLikeDecimalIntegerLiteral NonOctalDigit
-          NonOctalDecimalIntegerLiteral DecimalDigit
-
-        LegacyOctalLikeDecimalIntegerLiteral ::
-          `0` OctalDigit
-          LegacyOctalLikeDecimalIntegerLiteral OctalDigit
+      <p>The following syntax from <emu-xref href="#sec-literals-numeric-literals"></emu-xref>, and its associated semantics, used to be normative optional:</p>
```

Comment:
does it matter what it used to be?

Reply:
That's not my call. Like I said:
> I left a placeholder summary in B.1.1 to keep section numbers in sync, and in case a tombstone is wanted.

See [this comment](https://github.com/tc39/ecma262/issues/1595#issuecomment-504105211) and following.

Did the July meeting make a decision on what (if anything) is wanted here?

Reply:
It was not discussed, to my knowledge.